rcaracaus/pledge-vegan-node
# homepage h1 change for better seo

Steps to test:

1. Go to localhost:3000
2. Check that the home page has the title Pledge Vegan and that it has Pledge Vegan Blogs as a h1 at the top of the page
